test.sh

```diff
@@ -1,43 +1,24 @@
 #!/bin/bash -ex
 
-test_image=api-test
-docker build -t $test_image .
-
 function finish {
-	docker rm -f $pg_cid
-	docker rm -f $server_cid
+  docker-compose down
 }
 trap finish EXIT
 
-possum_tag=0.1.0-stable
-possum_image=registry.tld/possum:$possum_tag
-
-export POSSUM_DATA_KEY="$(docker run --rm $possum_image data-key generate)"
-
-pg_cid=$(docker run -d postgres:9.3)
-
-server_cid=$(docker run -d \
-	--link $pg_cid:pg \
-	-e DATABASE_URL=postgres://postgres@pg/postgres \
-	-e RAILS_ENV=test \
-	$possum_image server)
+# Generate reports folders locally
+mkdir -p spec/reports features/reports
 
-set -o pipefail
-admin_api_key=( $(cat ci/setup-account.sh | docker exec -i $server_cid /bin/bash | tail -1) )
+# Build test container & start the cluster
+docker-compose build
+docker-compose up -d
```

Comment:
Unfortunately, this command has a `--build` arg, but not a `--pull` arg, so it's a two-step process with line 12.